src/fides/api/schemas/saas/saas_config.py

```diff
@@ -223,10 +223,47 @@ def validate_override(cls, values: Dict[str, Any]) -> Dict[str, Any]:
         return values
 
 
+class ReadSaaSRequest(SaaSRequest):
+    """
+    An extension of the base SaaSRequest that allows the inclusion of an output template
+    that is used to format each collection result.
+    """
+
+    output: Optional[str]
```

Comment:
per comment below, just wondering if we should have some sort of validation here that this is a valid JSON template?